common/services/event.js

```diff
@@ -0,0 +1,85 @@
+const restClient = require('../lib/api-client/rest-client')
+
+const BaseService = require('./base')
+class EventService extends BaseService {
+  postEvents(lockoutEvents, move, user) {
+    const PER_EVENTS = [
+      'PerViolentDangerous',
+      'PerWeapons',
+      'PerConcealed',
+      'PerSelfHarm',
+      'PerEscape',
+      'PerMedicalAid',
+      'PerMedicalDrugsAlcohol',
+      'PerMedicalMentalHealth',
+      'PerPropertyChange',
+      'PerGeneric',
+    ]
+    const PERSON_MOVE_EVENTS = [
+      'PersonMoveUsedForce',
+      'PersonMoveDeathInCustody',
+    ]
```

Comment:
I'd be tempted to pull these out of the function and to the top of the file or in the classs.